package.json

```diff
@@ -102,15 +104,12 @@
     "three": "0.87.0",
     "typeahead.js": "^0.11.1",
     "typedoc": "^0.11.1",
-    "typescript": "^2.0.0",
     "typeface-roboto": "0.0.54",
     "underscore": "^1.8.3",
     "url-join": "4.0.0",
     "url-loader": "^0.5.8",
     "velocity.java": "^1.3.1",
     "webpack": "3.6.0",
-    "webpack-dev-server": "2.8.2",
```

Comment:
@filippomc this is not needed to use the webpack devserver i.e. npm start?

Reply:
Same as above for typescript. Should be a devDependency. I also opened an issue about that: https://github.com/openworm/geppetto-client/issues/42

Reply:
Let's have a separate PR for these then